webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -169,6 +169,16 @@ export default {
     },
     updateEditorContent(value) {
       // this.form.content = value
+      this.disabled = true
+      if (
+        value.replace(
+          /<p>|<\/p>|<h3>|<\/h3>|<h4>|<\/h4>|<ul>|<\/ul>|<ol>|<\/ol>|<li>|<\/li>|<blockquote>|<\/blockquote>|<strong>|<\/strong>|<em>|<\/em>|<a.*>|<\/a>/gm,
+          '',
+        ).length > 3
```

Comment:
Its' quite obscure what you do here. I guess you want to avoid validation errors in the backend? Do we even have a validation for the length of a `post.content`? @ogerly could you double-check?

Anyways, ideally, you would explain what you do here with a test case :+1:

Reply:
no the number of characters in a content is not limited . i would suggest 2000 characters. maybe 4000. you can say badly .. some like to write whole romanes. maybe there should be a BLOG corner for every user for something like that. 
i think 2000 characters is ok. those are 2 A4 pages

Reply:
no the number of characters in a content is not limited . i would suggest 2000 characters. maybe 4000. you can say badly .. some like to write whole romanes. maybe there should be a BLOG corner for every user for something like that. 
i think 2000 characters is ok. those are 2 A4 pages

Reply:
![FireShot Capture 174 - Human Connection - Human Connection - localhost](https://user-images.githubusercontent.com/1324583/60949675-f73a6f80-a2f5-11e9-95dc-49e72e7b9fd4.png)

Reply:
But I think we're not talking about maximum but *minimum* characters, right? Or how do you explain what your code does?